addons/website/static/src/builder/website_builder.js

```diff
@@ -69,6 +69,7 @@ export class WebsiteBuilder extends Component {
         builderProps.Plugins = Plugins;
         builderProps.onEditorLoad = (editor) => {
             this.editor = editor;
+            window.editor = editor; // For debugging purposes
```

Comment:
I don't have code like that for testing. It could create memory leaks as well.
I'll probably do a patch in the tour to expose the editor or to expose addStep. Maybe even have a patch just for specific tests.